tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: tutorial
+title: "International Flutter Apps"
+
+permalink: /tutorials/international/
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* How to track the device's locale (the user's preferred language).
+* How to manage locale-specific app values.
+* How to extend the locales that Flutter supports.
+
+</div>
+
+<a name="How To Write International Flutter Applications"></a>
+## How To Write International Flutter Applications
```

Comment:
We usually use a pattern of "____-ing something" for section titles. Such as "Internationalizing Flutter Applications"